static/js/components/inputs/SelectField.js

```diff
@@ -135,25 +109,30 @@ export default class SelectField extends React.Component {
     }
   };
 
-  autocompleteProps: Function = (): Object => {
+  getAutocompleteProps: Function = (): Object => {
     const {
+      autocompleteBehaviors,
+      autocompleteStyleProps,
       label,
       options,
       keySet,
       errors
     } = this.props;
+    // this.props.autocompleteBehaviors is passed in as a single function or a list of functions
+    let autocompleteProps = _.isFunction(autocompleteBehaviors) ?
+      autocompleteBehaviors(this.props) :
+      combineObjectArray(callFunctionArray(autocompleteBehaviors, this.props));
```

Comment:
It looks like `combineObjectArray` can be replaced with `Object.assign({}, ...callFunctionArray(autocompleteBehaviors, this.props))`